packages/d2c-schematics/workspace/files/next.config.js.template

```diff
@@ -15,6 +15,10 @@ const nextConfig = {
         protocol: "https",
         hostname: "**.cm.elasticpath.com",
       },
+      {
+        protocol: "https",
+        hostname: "cdn.shopify.com",
```

Comment:
This should not be included as part of the template, it's specific to the user/customer whos building a storefront.